homeassistant/components/broadlink/updater.py

```diff
@@ -57,13 +60,14 @@ class BroadlinkUpdateManager(ABC, Generic[_ApiT]):
 
     SCAN_INTERVAL = timedelta(minutes=1)
 
-    def __init__(self, device: BroadlinkDevice[_ApiT]) -> None:
+    def __init__(self, device: BroadlinkDevice[_ApiT], entry: ConfigEntry) -> None:
         """Initialize the update manager."""
         self.device = device
         self.coordinator = DataUpdateCoordinator(
             device.hass,
             _LOGGER,
             name=f"{device.name} ({device.api.model} at {device.api.host[0]})",
+            config_entry=entry,
```

Comment:
How about retrieving the config entry directly from `device.config`, avoiding the need for an additional parameter up to this point?

Reply:
Good point!